src/adapters/hasura/courseTracking.adapter.ts

```diff
@@ -0,0 +1,233 @@
+import { Injectable } from "@nestjs/common";
+import { HttpService } from "@nestjs/axios";
+import { SuccessResponse } from "src/success-response";
+import { CourseTrackingDto } from "src/courseTracking/dto/courseTracking.dto";
+
+@Injectable()
+export class CourseTrackingService {
+  constructor(private httpService: HttpService) {}
+  public async getCourseTracking(courseTrackingId: any, request: any) {
+    var axios = require("axios");
+
+    var data = {
+      query: `query GetCourseTracking($courseTrackingId:uuid) {
+      coursetracking(where: {courseTrackingId: {_eq: $courseTrackingId}}) {
+        contentIds
+        certificate
+        courseId
+        courseTrackingId
+        created_at
+        endTime
+        progressDetail
+        startTime
+        status
+        updated_at
+        userId
+        date
```

Comment:
@ManojNathIC Wy we are using date?

Reply:
Using date for course tracking current date